js/client/modules/@arangodb/testutils/crash-utils.js

```diff
@@ -69,8 +69,10 @@ let GDB_OUTPUT = '';
 function analyzeCoreDump (instanceInfo, options, storeArangodPath, pid) {
   let gdbOutputFile = fs.getTempFile();
 
-  let command = 'ulimit -c 0; (';
-  command += 'printf \'' +
+  let command = 'ulimit -c 0; sleep 10;(';
```

Comment:
Not sure why the `sleep 10` was necessary? Does ulimit take time to apply in the background?

Reply:
the sleep statements are here so that gdb can start up before sending commands to it. Loading coredumps can take some time; else it may miss the piped commands.